src/main/webapp/i18n/en/exercise-actions.json

```diff
@@ -46,6 +46,8 @@
             "cloneRatedRepository": "Clone your rated repository for this exercise:",
             "clonePracticeRepository": "Clone your practice repository for this exercise:",
             "cloneExerciseRepository": "Clone a repository for this exercise",
+            "waitForData": "Data is still being loaded. Please wait.",
+            "fetchVCSAccessTokenError": "Data could not be retrieved. Please reload the page and try again. If this issue persists, please contact an administrator.",
```

Comment:
This error message is too technical and vague. Can you try to describe it so an end user can understand it better?

Reply:
```suggestion
            "fetchVCSAccessTokenError": "Credentials to access the repository could not be loaded. Please reload the page and try again. If this issue persists, please contact an administrator.",
```